go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/openshift/ocm-agent
 
-go 1.19
+go 1.21
```

Comment:
There is a go version change here but the base image doesn't have required version https://github.com/openshift/ocm-agent/blob/master/build/Dockerfile#L1 

I have added a hold for now.. For quickest solution it might be ok to only change the go mod deps using command `go mod tidy -compat=1.19` and address the CVE.. But we will need separate PR for go version change along with testing and validation as well.. Thoughts?

Reply:
Good point @ravitri . We will create a card for bumping the go version later then..